src/Components/profile/RocketCard.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+function RocketCard() {
+  const { rockets } = useSelector((store) => store.rocket);
+  let counter = 0;
+  rockets.map((data) => {
+    if (data.reserved) {
+      counter += 1;
+    }
+    return counter;
+  });
```

Comment:
- [ ] Your app is working properly, nice job! However, it would be better to use `filter()` instead of `map()` to render the reserved rockets, as this is a better practice for JS/React.